README.md

```diff
@@ -111,7 +111,10 @@ class Cat
   end
 
   private
-  attr_accessor :age
+
+  def age
```

Comment:
I think we should keep this as a private `attr_accessor` (especially since private `attr_accessor`s are mentioned below). 

In #15, I meant more that we would have in another (public) method something like what you have here (it would have to be called something other than `age`, though -- as written, this would throw a `SystemStackError` (stack level too deep)).

Make sense?

```ruby
class Cat
  # ... all of the regular stuff

  def celebrate
    "Happy #{age}th birthday!"
  end

  private
  attr_accessor :age
end
```

As an aside, private `attr_accessor`s are a little funky, but I guess this is okay?